deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -102,6 +102,11 @@ spec:
             path: podStatus
             x-descriptors:
               - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
+          - displayName: Infinispan Console URL
```

Comment:
@dmvolod Is it possible to change the product name for downstream? s/Infinispan/Data Grid

Reply:
@oraNod seems to be we aren't changing another entrie names abd descriptions moving from upstream to downstream. I think it's possible but at the downstream site only.
@rigazilla wdyt?

Reply:
I think it's possible changing the CSV for Data Grid

Reply:
Yes, sure, thanks @rigazilla 
Do we need to change it now or can be done during product build?

Reply:
@dmvolod we will do on the product side